scripts/generate-constants.js

```diff
@@ -0,0 +1,84 @@
+const fs = require('fs');
+const path = require('path');
+
+// Read version from package.json
+const packageJsonPath = path.join(__dirname, '..', 'package.json');
+
+if (!fs.existsSync(packageJsonPath)) {
+  console.error(`Error: package.json not found at ${packageJsonPath}`);
+  process.exit(1);
+}
+
+let packageJson;
+try {
+  packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf8'));
+} catch (err) {
+  console.error(`Error: Failed to parse package.json: ${err.message}`);
+  process.exit(1);
+}
+
+const version = packageJson.version;
+if (!version) {
+  console.error('Error: No version found in package.json');
+  process.exit(1);
+}
+
+console.log(`Generating constants with version: ${version}`);
+
+// iOS template
+const iosTemplatePath = path.join(__dirname, '..', 'ios', 'RCTTWVideoConstants.h.template');
+const iosOutputPath = path.join(__dirname, '..', 'ios', 'RCTTWVideoConstants.h');
```

Comment:
Can be simplified a bit:
```suggestion
const iosOutputPath = path.join(path.dirname(iosTemplatePath), 'RCTTWVideoConstants.h');
```